recipes/LibriSpeech/ASR/transducer/hparams/conformer_transducer.yaml

```diff
@@ -399,3 +399,17 @@ error_rate_computer: !name:speechbrain.utils.metric_stats.ErrorRateStats
 
 cer_computer: !name:speechbrain.utils.metric_stats.ErrorRateStats
    split_tokens: True
+
+# for the inference hparams, you will need to include and uncomment something like this:
```

Comment:
Should we have a better documentation on how to turn a trained model to streaming? Like a colab?

Reply:
Right, but in the meantime, we might want to make it clear how to do this somewhere.

Reply:
Added as a notice in the README.